packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -28,7 +28,7 @@ import { copyReferencedFiles, updateLocalRelativeRefs } from './helpers/copy-ref
 import { generateOperationFinderFromSingleFile } from './helpers/path-extractor';
 
 const JAVA_OPTIONS = ['specPath', 'specConfigPath', 'globalProperty', 'outputPath'];
-const OPEN_API_TOOLS_OPTIONS = ['generatorName', 'output', 'inputSpec', 'config', 'globalProperty'];
+const OPEN_API_TOOLS_OPTIONS = new Set(['generatorName', 'output', 'inputSpec', 'config', 'globalProperty']);
```

Comment:
do we want that ?

Reply:
Indeed, it is faster in runtime but we way loose strong typing

Reply:
rule `unicorn/prefer-set-has`